NEWS.md

```diff
@@ -434,6 +434,9 @@ Deprecated or removed
   * `expand(ex)` and `expand(module, ex)` have been deprecated in favor of
     `Meta.lower(module, ex)` ([#22064, #24278]).
 
+  * `ones(A::AbstractArray[, opts...])` and `zeros(A::AbstractArray[, opts...])` methods
+    have been deprecated ([#24656]).
```

Comment:
in favor of `fill!(similar(A[, opts...]), 0/1)`?

Reply:
Good call! And done! Thanks! :)